finished/build.gradle

```diff
@@ -25,8 +25,8 @@ android {
         applicationId "com.example.android.wearable.composeforwearos"
         // 26 - Second oldest Wear 2.0 version (majority of devices)
         minSdk 26
-        // 30 - First Wear 3.0 version (and latest version of Wear OS)
-        targetSdk 30
+        // 33 Latest version of Wear OS
```

Comment:
This feels like it will be always getting out of date.

Maybe link to https://developer.android.com/google/play/requirements/target-sdk ?

Reply:
good idea! done.